src/index.ts

```diff
@@ -2,11 +2,14 @@ let totalPersonas: number = Number(
   prompt(`Ingrese cantidad de personas en la disco`)
 );
 let personas: number[] = new Array(totalPersonas);
-let i: number;
 let menores21: number = 0;
 let mayores21: number = 0;
 
-for (i = 0; i < totalPersonas; i++) {
+function aleatorio(min: number, max: number): number {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
```

Comment:
Las funciones (tambien las variables) deben ser declaradas antes de ser utilizadas.
Entonces la funcion **aleatorio** la movemos por encima del for que es donde se usa.